### ISSUE TITLE
[NBug] An error occurred trying to start process 'C:\Program Files...

### ISSUE THREAD

**Issue by KUppiano** [gitextensions/gitextensions#10467]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 6.0.11
- DPI 96dpi (no scaling)

### Issue description

Command: C:\Program Files (x86)\GitExtensions\PuTTY\pageant.exe

```
GitExtUtils.ExternalOperationException: An error occurred trying to start process 'C:\Program Files (x86)\GitExtensions\PuTTY\pageant.exe' with working directory 'C:\Program Files (x86)\GitExtensions'. The system cannot find the file specified.
 ---> System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\Program Files (x86)\GitExtensions\PuTTY\pageant.exe' with working directory 'C:\Program Files (x86)\GitExtensions'. The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at GitCommands.Executable.ProcessWrapper..ctor(String fileName, String prefixArguments, String arguments, String workDir, Boolean createWindow, Boolean redirectInput, Boolean redirectOutput, Encoding outputEncoding, Boolean useShellExecute, Boolean throwOnErrorExit)
   --- End of inner exception stack trace ---
   at GitCommands.Executable.ProcessWrapper..ctor(String fileName, String prefixArguments, String arguments, String workDir, Boolean createWindow, Boolean redirectInput, Boolean redirectOutput, Encoding outputEncoding, Boolean useShellExecute, Boolean throwOnErrorExit)
   at GitCommands.Executable.Start(ArgumentString arguments, Boolean createWindow, Boolean redirectInput, Boolean redirectOutput, Encoding outputEncoding, Boolean useShellExecute, Boolean throwOnErrorExit)
   at GitUI.CommandsDialogs.FormBrowse.StartAuthenticationAgentToolStripMenuItemClick(Object sender, EventArgs e)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

I reinstalled Git Extensions to obtain the update because it wouldn't upgrade from my previous version. Now when I attempt to start PuTTY, or pageant.exe, it is not installed. Setup doesn't prompt me for SSH Client. I am completely dead in the water.

### Steps to reproduce

Simply install this version of Git Extensions and attempt to load SSH key or go to Tools/PuTTY/Start authentication agent. An error will ensue. I encountered this while trying to submit changes to our Git repository, and failed to authenticate. I cannot do my work.

I looked for documentation online for repairing the problem. Repair options in the installer are not available for SSH or PuTTY.

### Did this work in previous version of GitExtensions?

Yes, but I can't remember what version I was using. I uninstalled it in order to install this version, because the upgrader said it couldn't upgrade the previous version. So we weren't in a good start to begin with. But the installer never gave me the option to select or configre an SSH client. The directory C:\Program Files (x86)\GitExtensions\PuTTY\ is missing.

**Comment by KUppiano** [comment]
Update: I found previous bugs which indicate that PuTTY is no longer installed with this product. I think the error messaging could be more helpful, either by linking to online docs, or by explicitly addressing the specific problem. 

I was able to repair my setup by manually tracking down and installing PuTTY, and then going to Git Extension Tools / Settings, and selecting the SSH node in the left pane. I had to switch from OpenSSH (which was selected by default) to PuTTY. The correct paths to the PuTTY executables were already correctly set. I applied my changes, and the feature began working correctly.

**Comment by KUppiano** [comment]
I was able to resolve the problem by reading comments on related bugs.

**Comment by RussKie** [comment]
Duplicate of #10321

**Comment by RussKie** [comment]
Fixed in  #10357
